contributing-tips.rst

```diff
@@ -0,0 +1,142 @@
+Tips for Making Good Contributions
+====================================
+
+.. _small-pr:
+
+Smallest meaningful PR
+------------------------
+
+A PR should normally address one issue. This makes it easier to review, easier to deploy, and easier to roll back in case of a problem. Additionally, the smaller the PR, the less likely it is to create a merge issue.
+
+The exception is when several issues are closely related or can reasonably be worked on together.  In this case, it should be clear by looking at the conversation on the Github issues that the items are related and will be worked on together.
```

Comment:
There can be a point about several issues linked together. Like *Best Practices/Tips issue* and *Adding Android level security tips* are interconnected. It's possible a PR can address one issue completely and partly resolve other. How can that be addressed while describing the PR?

Reply:
I'll add a sentence about making it clear in the PR as well as the issues.